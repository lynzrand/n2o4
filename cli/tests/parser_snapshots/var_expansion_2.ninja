# From: Ninja doc, "Lexical Syntax"
spaced = foo bar

rule foo
    command = command $out

build $spaced/baz other$ file: foo
