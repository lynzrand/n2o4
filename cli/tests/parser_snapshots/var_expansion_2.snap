Ok(
    NinjaFile {
        global_scope: {
            "spaced": "foo bar",
        },
        rules: {
            "foo": Rule {
                vars: {
                    "command": Expandable(
                        [
                            Regular(
                                "command ",
                            ),
                            Var(
                                "out",
                            ),
                        ],
                    ),
                },
            },
        },
        builds: [
            Build {
                inputs: [],
                implicit_inputs: [],
                order_only_inputs: [],
                outputs: [
                    "foo bar/baz",
                    "other file",
                ],
                command: "command 'foo bar/baz' 'other file'",
                depfile: None,
                deps: None,
                msvc_deps_prefix: None,
                description: None,
                dyndep: None,
                generator: false,
                restat: false,
                rspfile: None,
                rspfile_content: None,
            },
        ],
    },
)
